contracts/registry/JurorsRegistry.sol

```diff
@@ -563,7 +561,7 @@ contract JurorsRegistry is ControlledRecoverable, IJurorsRegistry, ERC900, Appro
             return 0;
         }
 
-        // TODO: Can this calculation be minimised?
+        // TODO: Can this calculation be minimised considering it uses the jurors token total supply twice?
```

Comment:
Last two lines think are equivalent to 

```
    jurorsTokenTotalSupply.mul(maxMaxPctTotalSupply).sub(totalActiveBalanceAtTerm.mul(diffOfPct)) / PCT_BASE;
```

but may make sense to leave as is for better readability.

Reply:
Cheers man, yeah will leave as is.